pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_property_declaration.dart

```diff
@@ -0,0 +1,87 @@
+import '../../../ast/_core/interfaces/declaration.dart';
+import '../../../ast/_core/shared/referred_type.dart';
+import '../../../ast/declarations/compounds/class_declaration.dart';
+import '../../_core/json.dart';
+import '../../_core/parsed_symbolgraph.dart';
+import '../../_core/utils.dart';
+import '../parse_declarations.dart';
+
+ClassPropertyDeclaration parsePropertyDeclaration(
+  Json propertySymbolJson,
+  ParsedSymbolgraph symbolgraph,
+) {
+  return ClassPropertyDeclaration(
+    id: parseSymbolId(propertySymbolJson),
+    name: parseSymbolName(propertySymbolJson),
+    type: _parsePropertyType(propertySymbolJson, symbolgraph),
+    hasObjCAnnotation: symbolHasObjcAnnotation(propertySymbolJson),
+    hasSetter: _propertyHasSetter(propertySymbolJson),
+  );
+}
+
+ReferredType _parsePropertyType(
+  Json propertySymbolJson,
+  ParsedSymbolgraph symbolgraph,
+) {
+  final subHeadings = propertySymbolJson['names']['subHeading'];
+
+  final typeSymbolJson =
+      subHeadings.firstJsonWhereKey('kind', 'typeIdentifier');
+  final typeSymbolId = typeSymbolJson['preciseIdentifier'].get<String>();
+  final typeSymbol = symbolgraph.symbols[typeSymbolId];
+
+  if (typeSymbol == null) {
+    throw Exception(
+      '''The property at path "${propertySymbolJson.path}" has a return type that does not exist among parsed symbols.''',
```

Comment:
You only need to use `'''` for multi-line strings. If you're just writing a string that contains `"`, you can use `'`. Also, you can break up this string to [adjacent string literals](https://dart.dev/language/built-in-types#strings:~:text=adjacent%20string%20literals). That way the literals can be placed on separate lines, and you can stay under the 80 char width limit.

Reply:
I was using `'''` cause it silences the 80 char width limit warning. Didn't know about adjacent strings syntax.
I tracked all such occurrences and replaced them with adjacent strings.